app/assets/stylesheets/trips.css.scss

```diff
@@ -1,8 +1,16 @@
 #map{
+  border-radius: 5px 5px 5px 5px;
+  border: solid 2px #363636;
```

Comment:
Color literals like `#363636` should only be used in variable declarations; they should be referred to via variable everywhere else.